packages/stores/src/providers/WebsiteStatusProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+import { observer } from 'mobx-react-lite';
+import { useWS } from '@deriv/shared';
+import merge from 'lodash.merge';
+
+const WebsiteStatusProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, is_subscribed, subscribe, unsubscribe } = useSubscription('website_status');
+    const {
+        client: { is_logged_in, is_logging_in, loginid },
+        website_status: { update },
+    } = useStore();
+    const WS = useWS();
+
+    React.useEffect(() => {
+        if (data) {
+            if (data) update(prev => merge(prev, data));
+        }
+    }, [update, data]);
+
+    React.useEffect(() => {
+        // handles case where the response from website_status is different when client is logged in/out, we need to re-subscribe in that case
+        // refer to socket-general.js
+        if (is_logged_in || is_logging_in) {
+            if (is_subscribed) unsubscribe();
+            WS.get()
+                .expectResponse('authorize')
+                .then(() => {
+                    subscribe();
+                });
+        } else {
+            subscribe();
```

Comment:
there is a case where `p2p_config` field is not available when the user is logged out. In order for `website_status` to reflect this, at the moment we have to re-subscribe to the `website_status` to reflect this change, the same goes for user that is logged in.